### PR TITLE
Add retry on EPIPE error

### PR DESCRIPTION
**Pull request by southpolesteve**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "mocha.files.glob":"test/legacy/**/*.js",
     "editor.formatOnSave": true,
-    "typescript.tsdk": "node_modules/typescript/lib"
+    "typescript.tsdk": "node_modules/typescript/lib",
+    "workbench.colorCustomizations": {}
```
Comment (southpolesteve) [inline .vscode/settings.json]: vscode wants to add this. idk why

Comment (christopheranderson) [reply .vscode/settings.json]: 👀👀👀 y tho?

File: src/retry/defaultRetryPolicy.ts [hunk 2]
```diff
@@ -74,6 +74,12 @@ const WindowsNoRouteTohost = 10065;
  */
 const LinuxConnectionReset = "ECONNRESET";
 
+// Node Error Codes
+/**
+ * @ignore
+ */
+const BrokenPipe = "EPIPE";
```
Comment (christopheranderson) [inline src/retry/defaultRetryPolicy.ts]: something something test coverage

**Comment by southpolesteve** [comment]
Fixes #399

**Pull request merged**